setup.py

```diff
@@ -8,9 +9,10 @@
       packages=['pylgbst', "pylgbst.comms"],
       requires=[],
       extras_require={
-          'gatt': ["gatt"],
+          # Note that dbus and gi are normally system packages
```

Comment:
Please do regular formatting, like standards require

Reply:
Didn't realise you had a formatting spec. I don't see one in the repository, so I'm assuming black, though that seems to be producing rather large diffs on the modules to which I'm applying it.